packages/mdc-list/foundation.ts

```diff
@@ -649,6 +655,25 @@ export class MDCListFoundation extends MDCFoundation<MDCListAdapter> {
     this.focusedItemIndex = index;
   }
 
+  private toggleAll_(currentlySelectedIndexes: number[]) {
```

Comment:
Remove method trailing underscore. Even though some private methods have the trailing underscore we are moving towards not using them for private methods.